modules/user-guide/pages/analyze-vale-app.adoc

```diff
@@ -0,0 +1,29 @@
+// Metadata for Antora
+:navtitle: Analyzing with the Red Hat Vale app
+:keywords: vale, app
+:description: Describes how to quickly get started with Vale and analyze a batch of content by using the Vale app.
+:page-aliases: end-user-guide:running-vale-cli.adoc
+// End of metadata for Antora
+// Metadata for Modular Docs
+:context: assembly_getting-started-with-vale
+:_module-type: PROCEDURE
+// End of metadata for Modular Docs
+[id="proc_linting-vale-cli_{context}"]
+= Analyzing your content by using the Red Hat Vale web app
+
+If you are new to Vale and want to quickly get started without installing the client and prerequisites, use the web-based version of Vale. The Red Hat web app is preconfigured to analyze your content by using Red Hat style rules. The `vale-at-red-hat`` rules are built in and there is no need to install and maintain the Vale client and the `vale-at-red-hat`` style rules. 
+
+With the web app, you can run Vale on one source file at a time. You can also analyze a section of a content file.
+
+* To use the Vale starter app, enter https://vale.vercel.app/ in any browser, paste your AsciiDoc content, and click *Lint*.
+
+* By default, the Red Hat Vale web app displays warnings and errors only. You can also display suggestions by using options menu on the left. 
+
+////
+
+[IMPORTANT]
+====
+Add disclaimer about the support of the app here.
```

Comment:
@rolfedh - Do you have any suggestions for providing guidance about the supportability of the app?